WebApiCoreSeed.WebApi.Tests/Controllers/UserControllerTest.cs

```diff
@@ -0,0 +1,260 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using Moq;
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using WebApiCoreSeed.Data.Models;
+using WebApiCoreSeed.Domain.Services.Interfaces;
+using WebApiCoreSeed.WebApi.Controllers;
+using WebApiCoreSeed.WebApi.Models;
+using Xunit;
+
+namespace WebApiCoreSeed.WebApi.Tests.Controllers
+{
+    public class UserControllerTest
+    {
+        #region Fields
```

Comment:
Let's don't contaminate the seed with heresy 🤣 -- let's each team (users of the seeder) decide if they will use regions or not and keep this as pristine as possible.

Reply:
I know, I was opposed to add that section if remember correctly 😄 .